plugin/src/generators/repo-config/files/README.md.template

```diff
@@ -65,6 +56,60 @@ npx nx g react-lib shared/ui/ui-kit
 
 - `npx nx g expo-app` - generate new RN Expo app
 
+#### Rename library
```

Comment:
I would just write a link to public readme (npm or github)

Reply:
It's not very convenient

Reply:
@ipakhomov what do you think?